spacy_llm/tests/pipeline/test_llm.py

```diff
@@ -290,3 +290,20 @@ def test_pipe_labels():
         nlp.to_disk(tmpdir / "tst.nlp")
         nlp = spacy.load(tmpdir / "tst.nlp")
         assert nlp.pipe_labels["llm"] == ["COMPLIMENT", "INSULT"]
+
+
+def test_llm_task_factories():
```

Comment:
This is a good test, but I'd prefer having a few specific ones, e.g. have a test that adds an `llm_ner` component (one test from config, one test in code), maybe specifies an actual model, and checks that it actually runs properly.

Reply:
Added an `llm_ner` test in https://github.com/explosion/spacy-llm/pull/243/commits/e936e278ee2582f522993af0ad983a1c52f6cbde. Any other specific tests you'd want to have in there?